cluster/gce/util.sh

```diff
@@ -869,6 +853,30 @@ function kube-push {
   echo
 }
 
+function kube-update-nodes() {
+  action=${1}
+
+  OUTPUT=${KUBE_ROOT}/_output/logs
+  mkdir -p ${OUTPUT}
+
+  echo "Updating node metadata... "
+  write-node-env
+  for (( i=0; i<${#MINION_NAMES[@]}; i++)); do
+    add-instance-metadata-from-file "${MINION_NAMES[$i]}" "kube-env=${KUBE_TEMP}/node-kube-env.yaml" &
```

Comment:
BTW, I'm not sure what you meant by the node-kube-env.yaml here. I think we absolutely need to run the configure-vm from the right version, but I think for the time being the answer is that the configure-vm script of the future needs to consume the kube-env of the past. Having past code have to write out future bits is going to result in some sort of schema rewriting confusion that... nevermind. Let's chat on Monday. I threw a TODO in about the configure-vm script at least.